### PR TITLE
Feature/ns timeline event

### PR DESCRIPTION
**Pull request by RobTobias123**
Draft of ns-timeline documentation.

### CODE REVIEW

File: docs/components/ns-timeline-event.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+> The Timeline Event component represents a sequence of events that happen within a process, and shows the status.
+
+It gives an overview of the whole process and progress of a customer journey. Its main purpose is to create situational awareness for the user and manage expectations.
+
+* Provides the customer with visibility on the status of a process or system in their journey
+
+* Gives a top-level overview of the process
+
+* It's clear to understand and mangages expectations by showing the current situation to the user
```
Comment (PtitBen) [inline docs/components/ns-timeline-event.md:19]: manages

File: docs/components/ns-timeline-event.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+## Example (sped up)
+![example-ani-480](https://user-images.githubusercontent.com/78355810/122419834-20a0b300-cf83-11eb-927f-86ba856fe103.gif)
+
+## Content guidance
+
+### Status
+
+Each event has a status. The status of the event is controlled by the decoration used.
+
+| Style | Status | Description |
```
Comment (PtitBen) [inline docs/components/ns-timeline-event.md:36]: image
It looks like some of the status circles are different sizes?

File: docs/components/ns-timeline-event.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+#### Numbers 
+For use when an order should be shown.
+
+
+Some icons are special and automatically reserved for use only with a particular status, such as Complete tick, Action required arrow, Warning triangle, and Error diamond. These show on all the variants of 'overview decoration'. 
+
+### Heading, Summary and Event count
+
+There is provision for a Heading to describe what process the timeline is showing. This can be left blank if it has already been made clear in an ns-landmark above it to avoid duplication.
```
Comment (PtitBen) [inline docs/components/ns-timeline-event.md:63]: do we need to capitalise the h in heading?

**Review by PtitBen — COMMENTED** [review]
Looking good! 
As Dan mentioned, maybe explore a video instead of a GIF to make it lighter and potentially higher resolution in the process :) 

A few comments on tiny details in the review

File: docs/components/ns-timeline-event.md (hunk 2 above)
Comment (RobTobias123) [reply docs/components/ns-timeline-event.md:40]: That's correct. They are larger when 'in-progress' to have form change as well as colour to identify the step you are on. Action, warning and error states are also all 'in-progress' so appear at the larger size. In the component they align centrally but we don't have that option as far as I'm aware in the md?

File: docs/components/ns-timeline-event.md (hunk 1 above)
Comment (RobTobias123) [reply docs/components/ns-timeline-event.md]: Good spot - thanks!

File: docs/components/ns-timeline-event.md (hunk 3 above)
Comment (RobTobias123) [reply docs/components/ns-timeline-event.md]: I've removed the capitals for the 3 knobs and slightly improved the wording to be clearer around the naming.

File: docs/components/ns-timeline-event.md (hunk 2 above)
Comment (PtitBen) [reply docs/components/ns-timeline-event.md:39]: Ah, OK that's what threw me off, ignore this comment then :)

File: docs/components/ns-timeline.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+## Example (sped up)
+![example-ani-480](https://user-images.githubusercontent.com/78355810/122419834-20a0b300-cf83-11eb-927f-86ba856fe103.gif)
+
+Temporary video testing here - IGNORE - will be removed/replaced...
+<video controls src="https://user-images.githubusercontent.com/78355810/122530664-a15cbf00-d016-11eb-9875-ffbe2cf6a50a.mp4" type="video/mp4"></video>
+
+<figure class="video_container">
+  <video controls="true" allowfullscreen="true" poster="path/to/poster_image.png">
+    <source src="https://user-images.githubusercontent.com/78355810/122530664-a15cbf00-d016-11eb-9875-ffbe2cf6a50a.mp4" type="video/mp4">
```
Comment (jholt1) [inline docs/components/ns-timeline.md:35]:
```suggestion
    <source src="https://user-images.githubusercontent.com/78355810/122530664-a15cbf00-d016-11eb-9875-ffbe2cf6a50a.mp4" type="video/mp4" />
```

File: docs/components/ns-timeline.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+It gives an overview of the whole process and progress of a customer journey. Its main purpose is to create situational awareness for the user and manage expectations.
+
+* Provides the customer with visibility on the status of a process or system in their journey.
+* Gives a top-level overview of the process.
+* It's clear to understand and manages expectations by showing the current situation to the user.
+
+## Content guidance
+
+(Sequence shortened.)
+<!--![example-ani-480](https://user-images.githubusercontent.com/78355810/122419834-20a0b300-cf83-11eb-927f-86ba856fe103.gif)-->
```
Comment (dbenmore) [inline docs/components/ns-timeline.md:22]: lets delete this

**Review by PtitBen — APPROVED** [review]

File: docs/components/ns-timeline.md (hunk 5 above)
Comment (andij) [reply docs/components/ns-timeline.md:22]: Yeah, good shout.

**Pull request merged**